ndc-test/src/lib.rs

```diff
@@ -160,9 +164,14 @@ pub async fn test_snapshots_in_directory_with<
                                 .map_err(Error::CannotOpenSnapshotFile)?;
                             let request = serde_json::from_reader(request_file)?;
 
-                            let response = f(request).await?;
+                            let response = f(request).await;
+
+                            match response {
+                                Ok(response) => snapshot_test(snapshot_path, &response),
+                                Err(e) => Err(e),
+                            }
 
-                            snapshot_test(snapshot_path, &response)
+                            // snapshot_test(snapshot_path, &response)
```

Comment:
Did you mean to leave this in?

Reply:
No, it's a stray comment now. I will remove it.